region/crateria/east.json

```diff
@@ -76,7 +76,23 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": []
+                  "requires": ["canDisableEquipment"]
```

Comment:
```suggestion
                  "requires": ["canDisableEquipment"],
                  "note": "This jump is much easier without HiJump or Speedbooster equipped."
```
It's possible to get the item with a careful/tricky jump, but I think this is a fine way to represent it.